### PR TITLE
Adding a community extension draft

### PR DESCRIPTION
**Pull request by jaygambetta**
This is a proposal for the community extensions page. 

Looks like 

image

### CODE REVIEW

**Review by atilag — DISMISSED** [review]

File: docs/community.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Qiskit Community Extensions
+===========================
+
+Qiskit has been designed to modular in nature. It is extendable in many different ways and
```
Comment (atilag) [inline docs/community.rst]: ... designed **with modularity in mind**. It's extendable...  ?

**Review by atilag — DISMISSED** [review]

File: docs/community.rst (hunk 1 above)
Comment (derivation) [reply docs/community.rst]: +1 on "...designed with modularity in mind."

Keep "It is" to stay formal.

"extensible" is more common in this use case than "extendible".

**Review by derivation — DISMISSED** [review]
I approve. My in-line comments are just suggestions.

**Comment by jaygambetta** [comment]
could you make the changes they are good.

**Comment by nonhermitian** [comment]
I think we should explicitly make clear the qiskit versions that the extensions work for.

**Review by derivation — APPROVED** [review]

**Comment by jaygambetta** [comment]
@nonhermitian that is a good idea but that should go in the bullets like 

**qiskit version** 0.7

**Comment by derivation** [comment]
@nonhermitian good idea re: bullets of Qiskit versions. Noted for when projects start getting added to the list.

**Pull request merged**